### PR TITLE
Feat/base error v2 initial implementations

### PR DESCRIPTION
**Pull request by TheHussein**

### CODE REVIEW

**Review by MaxAndDev — APPROVED** [review]
LGTM

File: src/entities/person/channel-user.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import Entity, { UniverseEntityOptions, EntityRawPayload, EntityFetchOptions, UniverseEntity } from '../_base'
+import { UniverseEntityOptions, EntityRawPayload, EntityFetchOptions, UniverseEntity } from '../_base'
 import { Universe } from '../../universe'
 import * as messageTemplate from '../message-template/message-template'
 import * as event from '../../eventing/feeds/event'
 import * as feed from '../../eventing/feeds/feed'
-import { BaseError } from '../../errors'
+import { BaseErrorV2, BaseErrorV2Properties } from '../../errors'
 import qs from 'qs'
+import { AxiosError } from 'axios'
```
Comment (eljefedelrodeodeljefe) [inline src/entities/person/channel-user.ts:8]: nit: `import type { AxiosError } from 'axios'` should be enough

**Review by eljefedelrodeodeljefe — CHANGES_REQUESTED** [review]
See comments above

**Comment by eljefedelrodeodeljefe** [comment]
@TheHussein added some fixes and tests that were necessary. PTAL and then let's go ahead with it.

**Review by eljefedelrodeodeljefe — APPROVED** [review]

**Pull request merged**